lib/compareLocations.js

```diff
@@ -25,9 +33,10 @@ module.exports = (a, b) => {
 				if (ap.line > bp.line) return 1;
 				if (ap.column < bp.column) return -1;
 				if (ap.column > bp.column) return 1;
+			} else if (a.index && b.index) {
+				if (a.index < b.index) return -1;
+				if (a.index > b.index) return 1;
 			}
-			if (a.index < b.index) return -1;
```

Comment:
@sokra for HarmonyExportDependecy, .index is patched to this source location instance. Technically this would throw if used elsewhere no? How should I guard from this.

Reply:
I ask cause it is causing unittest failures